src/build/build.ts

```diff
@@ -13,14 +13,16 @@
  */
 
 import * as bower from 'bower';
+import * as gulpif from 'gulp-if';
 import * as path from 'path';
 import * as logging from 'plylog';
 import {dest} from 'vinyl-fs';
 
 import mergeStream = require('merge-stream');
 import {forkStream, PolymerProject, addServiceWorker, SWConfig, HtmlSplitter} from 'polymer-build';
 
-import {OptimizeOptions, getOptimizeStreams} from 'polymer-build';
+import {getOptimizeStreams} from 'polymer-build';
+import {JsTransform, HtmlTransform, matchesExt} from 'polymer-build/lib/optimize-streams';
```

Comment:
Maybe we should re-export these from polymer-build

Reply:
I'm not all that excited about adding them to our public API surface.